src/Card/CardSection.jsx

```diff
@@ -45,7 +45,7 @@ CardSection.propTypes = {
   /** Specifies class name to append to the base element. */
   className: PropTypes.string,
   /** Specifies contents of the component. */
-  children: PropTypes.node.isRequired,
+  children: PropTypes.node,
```

Comment:
turns out it is a valid use case to use `Card.Section` without `children`

Reply:
Love the built in spacers of the `Card.Section` :)